json/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "bitcoincore-rpc-json"
-version = "0.16.0"
+version = "1.0.0-rc0"
```

Comment:
This looks quite bold considering crazy-new change and the amount of breaking changes at Core.

Reply:
Yeah version is a draft. Can be anything, but I'd like to be able to release rc's while also still being able to patch the current branch. So either it would be 0.20 or something or 1.0. Doesn't have to be too bold, we can go until 2^64-1 probably with versions ;)

Reply:
1.0+ means there should be minimal if any breaking changes.